tests/test_orm_mode.py

```diff
@@ -305,15 +306,14 @@ class Config:
 
 
 def test_recursive_parsing():
-    from types import SimpleNamespace
-
     class Getter(GetterDict):
         # try to read the modified property name
         # either as an attribute or as a key
         def get(self, key, default):
             key = key + key
             try:
-                return self._obj[key]
+                v = self._obj[key]
+                return Getter(v) if isinstance(v, dict) else v
```

Comment:
@nuno-andre Would it make sense to you?

Reply:
I'm afraid I don't get the point. Wouldn't this change break the test?